src/runtime/mod.rs

```diff
@@ -83,7 +88,13 @@ impl Runtime {
         })
     }
 
-    /// Runs a future to completion on the current runtime
+    /// Runs a future to completion on the current runtime.
+    ///
+    /// Returns a Future that resolves to the output of the
+    /// supplied future.
+    ///
+    /// Note: Unlike most [`Futures`](std::future::Future), this future will
```

Comment:
This `Note` also strikes me as kind of opposite the kind of advice `Tokio` would give about its `block_on`. It should be understood that calling `block_on` is a task operation.

Also, which `this future` is being referred to? If such a note really is worthwhile, it should probably leave no room for doubt.

In fact, if we want to clean this public function documentation, it should be written as carefully as the `Tokio runtime block_on`.  Or just reference it and say what our wrapping of it accomplishes. https://github.com/tokio-rs/tokio/blob/048049f8883f1bc73ee5d3fa3c94e61cf41d1e38/tokio/src/runtime/runtime.rs#L223-L252

Reply:
>  Here, I agree it is warranted that it says a future is being returned.

Actually. I'm wrong.   That function is not async.  It doesn't return a future.  

I think the most accurate documentation would be to copy paste *most of* the tokio docs you reference, (omitting the multithreaded runtime bits)

Reply:
Looking more closely at the comments,  neither the `Current thread scheduler` or `Multi thread scheduler` sections are correct for tokio-uring. 

I'll leave them out for now, and in the future, someone with more insight into its precise behavior can comment.